library_panel.go

```diff
@@ -101,22 +101,30 @@ func (c *Client) LibraryPanels() ([]LibraryPanel, error) {
 
 // LibraryPanelByUID gets a library panel by UID.
 func (c *Client) LibraryPanelByUID(uid string) (*LibraryPanel, error) {
-	return c.panel(fmt.Sprintf("/api/library-elements/%s", uid))
+	resp := &LibraryPanelCreateResponse{}
+	path := fmt.Sprintf("/api/library-elements/%s", uid)
+
+	err := c.request("GET", path, nil, nil, &resp)
+	if err != nil {
+		return nil, err
+	}
+
+	return &resp.Result, err
 }
 
-// LibraryPanelByName gets a library panel by name.
+// LibraryPanelByName gets library panels by name.
 func (c *Client) LibraryPanelByName(name string) (*LibraryPanel, error) {
-	return c.panel(fmt.Sprintf("/api/library-elements/name/%s", name))
-}
+	resp := &struct {
+		Result []LibraryPanel `json:"result"`
+	}{}
```

Comment:
The more idiomatic way of writing this would be:
```suggestion
	var resp struct {
		Result []LibraryPanel `json:"result"`
	}
```